src/app/shared/model/audio-version.model.ts

```diff
@@ -39,13 +39,13 @@ export class AudioVersionModel extends BaseModel {
     this.setLabel();
   }
 
-  setLabel() {
+  private setLabel() {
     if (this.doc && this.doc['label']) {
-      this.set('label', this.doc['label']); // probably already set to this
+      this.label = this.doc['label']; // probably already set to this
```

Comment:
Yes, this triggering changed was causing problems. When the story audio version explicit was set to what the series had on a new draft story, it was showing up as changed because of the label (but none of the fields were highlighted orange because it was not explicit that had actually "changed" just had a value assigned as a default.)